neurokit2/ecg/ecg_findpeaks.py

```diff
@@ -105,12 +97,12 @@ def _ecg_findpeaks_findmethod(method):
         return _ecg_findpeaks_rodrigues
     elif method in ["vg", "vgraph", "koka2022"]:
         return _ecg_findpeaks_vgraph
+    elif method in ["FastNVG", "fastnvg", "emrich", "emrich2023"]:
```

Comment:
`method = method.lower()` above so you don't need the upper case variant :)

Reply:
True. I have fixed it in [7674b25](https://github.com/neuropsychology/NeuroKit/pull/939/commits/7674b25f5b3d5d44f4c6cad82c77f3531d9c9588)